Showcase-iOS/About/New Group1/AboutViewModel.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  AboutViewModel.swift
+//  Showcase-iOS
+//
+//  Created by Sashen Pillay on 2019/03/14.
+//  Copyright © 2019 DVT. All rights reserved.
+//
+
+import Foundation
+
+class AboutViewModel {
+
+    var socialMediaLinks: SocialMediaLinks?
+
+    var twitter: URL {
+        let url = URL(string: socialMediaLinks?.twitter ?? "https://twitter.com/dvt_corporate")
+        return url!
```

Comment:
This should not be forced unwrapped if you are doing the null check above. For all of these below. Please check

Reply:
done